### ISSUE TITLE
Silently errors, blocks webpack build on including html file

### ISSUE THREAD

**Issue by AndrewRayCode** [skozin/webpack-path-rewriter#2]
webpack.config.js:

```
module.exports = {
  entry: {
    app: "./app/index.jsx",
  },
  output: {
    path: "public",
    filename: "[name].js",
    publicPath: "/"
  },
  module: {
    loaders: [
      {test: /\.html$/, loader: PathRewriterPlugin.rewriteAndEmit({
        name: '[path][name].html',
        loader: 'file'
      })},
    ]
  },
  plugins: [
    new PathRewriterPlugin(),
  ]
};
```

app/index.js:

```
require("../assets/index.html");
```

assets/index.html:

```
<!DOCTYPE html>
<html>
  <head>
    <title>Empty</title>
  </head>
  <body></body>
</html>
```

When running webpack command line it silently errors, no output. The built files are not created. If I remove the require html line, it all works.

**Comment by AndrewRayCode** [comment]
ah, you never call `done` if there are no require paths https://github.com/skozin/webpack-path-rewriter/blob/master/es6/index.js#L245

**Comment by skozin** [comment]
@DelvarWorld, thanks for the report! The callback should definitely be called even if there are no paths. I'll fix this ~~in a couple of days~~ now.

**Comment by skozin** [comment]
Fixed in 1.1.1, you can re-install it now from npm =)
